src/posix/platform/infra_if.cpp

```diff
@@ -572,6 +572,63 @@ void InfraNetif::Update(Mainloop::Context &aContext)
 
 #ifdef __linux__
 
+void InfraNetif::ProcessNetLinkMessage(const struct nlmsghdr *aNetlinkMessage)
+{
+    const struct ifinfomsg *ifinfo = reinterpret_cast<struct ifinfomsg *>(NLMSG_DATA(aNetlinkMessage));
+
+    switch (aNetlinkMessage->nlmsg_type)
```

Comment:
RTM_NEWADDR and RTM_DELADDR are not handled now. I think it would cause problem when the address is added or removed.

Reply:
nit: may add comment to note down the intentions explicitly to avoid unexpected changes in the future